test/hotspot/jtreg/compiler/vectorapi/VectorCastShape128Test.java

```diff
@@ -42,6 +42,7 @@
  * @summary AArch64: 'bad AD file' in some vector conversion tests
  * @modules jdk.incubator.vector
  * @run testng/othervm -XX:-TieredCompilation compiler.vectorapi.VectorCastShape128Test
+ * @run testng/othervm -XX:-TieredCompilation -XX:+UnlockDiagnosticVMOptions -XX:+UseKNLSetting compiler.vectorapi.VectorCastShape128Test
```

Comment:
I suppose `UseKNLSetting` is a x86 specific option, is this okay for other platforms?